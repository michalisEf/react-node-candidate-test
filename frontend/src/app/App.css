.app {
  font-family: TotalSans, Arial, Helvetica, sans-serif;
  padding: 1rem;
}

