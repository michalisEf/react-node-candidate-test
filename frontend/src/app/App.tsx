import React from 'react';

import './App.css';

import { } from '../containers';

const App = () => {
	return (
		<div className="app">
			Start building your app here
		</div>
	);
}

export default App;