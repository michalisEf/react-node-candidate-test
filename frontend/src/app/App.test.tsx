import { render, screen } from '@testing-library/react';
import App from './App';

test('renders start text', () => {
  render(<App />);
  const app = screen.getByText(/Start building your app here/i);
  expect(app).toBeInTheDocument();
  expect(app).toMatchSnapshot();
});
