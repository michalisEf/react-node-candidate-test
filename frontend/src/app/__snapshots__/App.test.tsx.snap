// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders start text 1`] = `
<div
  class="app"
>
  Start building your app here
</div>
`;
